migration/pkg.go

```diff
@@ -0,0 +1,111 @@
+package migration
+
+import (
+	"encoding/json"
+	"fmt"
+	liberr "github.com/konveyor/controller/pkg/error"
+	"github.com/konveyor/controller/pkg/logging"
+	"github.com/konveyor/tackle2-hub/database"
+	"github.com/konveyor/tackle2-hub/model"
+	"gorm.io/gorm"
+)
+
+var log = logging.WithName("migration")
+
+const HubDBVersion = ".hub.db.version"
```

Comment:
Recommend this be (something like):
Version = "hub.migration.version" or just ".migration.version"